chia/full_node/mempool_manager.py

```diff
@@ -22,6 +22,14 @@
 from chia.full_node.pending_tx_cache import PendingTxCache
 from chia.types.blockchain_format.coin import Coin
 from chia.types.blockchain_format.sized_bytes import bytes32, bytes48
+from chia.types.borderlands import (
+    BlockRecordHeaderHash,
```

Comment:
mempool_manager.py was the main motivation to add these types differentiating various `byte32` types.